src/screencast/screencast.c

```diff
@@ -167,6 +168,13 @@ static int start_screencast(struct xdpw_screencast_instance *cast) {
 	wl_display_dispatch(cast->ctx->state->wl_display);
 	wl_display_roundtrip(cast->ctx->state->wl_display);
 
+	if (cast->screencopy_frame_info[WL_SHM].format == DRM_FORMAT_INVALID ||
```

Comment:
The SHM format should not be a DRM format? See `enum wl_shm_format`. 0 there means `WL_SHM_FORMAT_XRGB8888`, and `DRM_FORMAT_INVALID` also happens to be 0.

Reply:
We are using drm_format for everything internally, so this shouldn't be an issue. (see https://github.com/emersion/xdg-desktop-portal-wlr/blob/master/src/screencast/wlr_screencast.c#L106)

Reply:
Oh okay that makes sense then